index.html

```diff
@@ -23,6 +24,13 @@
       >
         Create Tasks
       </button>
+      <button
+        class="profile-task-btn"
+        type="button"
+        onclick="location.href='goal/index.html'"
+      >
+        Create Goals
+      </button>
```

Comment:
Why use a button to display links?

Reply:
Because other links like Create Task, Profiles are in button tag

Reply:
Can you create a ticket for this issue that to navigate from one page to another there should be anchor tag and this is the best practice ?

Reply:
done